changelog.d/11244.misc

```diff
@@ -0,0 +1 @@
+Rename MSC2716 related function to be more clear about the input and output (`get_insertion_event_id_by_batch_id`).
```

Comment:
It might be nice to use the same changelog as #11114 (or maybe reword it as part of that PR) so that towncrier combines them during release.

Reply:
It seems like the goal is to not burden the changelog with this nothing change. Which might be better suited to just not have a changelog entry for it.

Maybe a new changelog extension that we don't include in the changelog doc. Although `misc` feels pretty relevant for this kind of thing.

I'm kinda leaning towards leaving the changelog entry as-is.

Reply:
> It seems like the goal is to not burden the changelog with this nothing change. Which might be better suited to just not have a changelog entry for it.

This is why I suggested using the same one as #11114, they would then get combined into a single entry.

Reply:
Updated with same changelog ⏩

Not confident it staying the same in the other PR.

Reply:
> Not confident it staying the same in the other PR.

A thing we'll sometimes do is reword *other* changelogs in a PR to combine them.

Reply:
Ouu, good thinking! 😁

![](https://user-images.githubusercontent.com/558581/140858727-99216d9b-ba44-41a2-8ee7-23980f5fbf38.png)